frontend-react/src/App.tsx

```diff
@@ -128,6 +129,11 @@ const App = () => {
                                         authorize={PERMISSIONS.SENDER}
                                         component={Upload}
                                     />
+                                    <AuthorizedRoute
+                                        path="/submissions"
+                                        authorize={PERMISSIONS.PRIME_ADMIN}
+                                        component={Submissions}
+                                    />
```

Comment:
Unrelated but somewhat related, it appears that nav links aren't styled differently when active anymore. This might be solved when we're able to go to `react-router 6.x.x` since I know they deprecated `activeClassName` in favor of a new way of styling active links.

Reply:
I thought the react-router6 issue was related to okta's plugin that we use?
https://github.com/okta/okta-react/issues/178